soccer/gameplay/tactics/defense.py

```diff
@@ -78,435 +70,300 @@ def debug(self, value):
-                    # draw shot triangle
-                    pts = [threat.pos,
-                           threat.best_shot_window.segment.get_pt(0),
-                           threat.best_shot_window.segment.get_pt(1)]
-                    shot_color = (255, 0, 0, 150)  # translucent red
-                    main.system_state().draw_polygon(pts, shot_color,
-                                                     "Defense")
-                    main.system_state().draw_segment(
-                        threat.best_shot_window.segment, constants.Colors.Red,
-                        "Defense")
-
-                    self.win_eval.excluded_robots.clear()
-                    _, best_window = self.win_eval.eval_pt_to_our_goal(
-                        threat.pos)
-                    if best_window is not None:
-                        chance = best_window.shot_success
-                    else:
-                        chance = 0.0
-
-                    main.system_state().draw_text(
-                        "Shot: " + str(int(threat.shot_chance * 100.0)) +
-                        "% / " + str(int(chance * 100)) + "%",
-                        shot_line.center(), constants.Colors.White, "Defense")
-
-                # draw pass lines
-                if idx > 0:
-                    pass_line = robocup.Segment(main.ball().pos, threat.pos)
+                main.system_state().draw_line(shot_line, constants.Colors.Red,
+                                              "Defense-Shot Line")
+                main.system_state().draw_text(
+                    "Shot: " + str(int(shot_chance * 100.0)), threat[0],
+                    constants.Colors.White, "Defense-Shot Percent")
+
+                # Other threats besides ball
+                if threat_idx > 0:
+                    pass_line = robocup.Segment(main.ball().pos, threat[0])
                     main.system_state().draw_line(
-                        pass_line, constants.Colors.Red, "Defense")
-                    main.system_state().draw_text(
-                        "Pass: " + str(int(threat.ball_acquire_chance * 100.0))
-                        + "%", pass_line.center(), constants.Colors.White,
-                        "Defense")
+                        pass_line, constants.Colors.Red, "Defense-Pass Line")
+
+            # keep defenders from occupying the same spot
+            # only matters if there are 2 defenders (and the goalie)
+            if len(handlers) == 3:
+                handler1 = handlers[1]
+                handler2 = handlers[2]
+
+                #vector between the 2 points
+                overlap = handler2.move_target - handler1.move_target
+
+                #if the robots overlap
+                if overlap.mag() < (2 * constants.Robot.Radius) + .005:
+                    #move the robots away from each other
+                    overlap = overlap - (overlap.normalized() *
+                                         (constants.Robot.Radius + .1))
```

Comment:
Weren't you going to make this 0.1 into Radius * 2? Or was that not large enough?